aiida/backends/utils.py

```diff
@@ -38,7 +38,7 @@ def create_sqlalchemy_engine(profile, **kwargs):
         name=profile.database_name
     )
     return create_engine(
-        engine_url, json_serializer=json.dumps, json_deserializer=json.loads, encoding='utf-8', **kwargs
+        engine_url, json_serializer=json.dumps, json_deserializer=json.loads, future=False, encoding='utf-8', **kwargs
```

Comment:
Why do we need this `future=False` here? Would this break with v2.0?

Reply:
No actually this does not need to be there yet, its False by default; but moving to v2.0 API you set it to True, to make the switch (see https://github.com/aiidateam/disk-objectstore/pull/114 for the process)

Reply:
OK, so adding this here is not really necessary then? Since you are sticking with the v1.4 API here. Anyway, fine to keep as is, it is not that important. Was just wondering.